### DISCUSSION TITLE
Getting an error on Nuget Pack when updating to latest Uno Platform

### DISCUSSION THREAD

**Discussion by assassin316** [unoplatform/uno#11994] [Q&A] [answered]
I'm updating my library to the latest Uno version. When executing a nuget pack command, I get the following:

` NU1012: Some included files are included under TFMs which are missing a platform version: net6.0-android, net6.0-ios`
 
When upgrading, I had to update my nuspec from:

```
 <files>
    <file src="bin\Release\monoandroid10.0\*.*" target="lib\monoandroid10.0" />
    <file src="bin\Release\monoandroid11.0\*.*" target="lib\monoandroid11.0" />
    <file src="bin\Release\xamarinios10\*.*" target="lib\xamarinios10" />
 </files>
```
 
to:

```
 <files>
    <file src="bin\Release\net6.0-android\*.*" target="lib\net6.0-android" />
    <file src="bin\Release\net6.0-ios\*.*" target="lib\net6.0-ios" />
 </files>
```
 
I'm assuming I need to install some new packages locally?

**Comment by jeromelaban** [comment, marked as answer]
This is a curious error. Are you using net6.0 targets explicitly? In general, building nuget packages is generally easier to do without nuspec files and use msbuild targets.

**Reply by assassin316** [reply]
MSBuild  did the trick, I'll move the rest over to MSBuild and scrap nusepc. Thanks for the quick reply!
